src/org/traccar/web/CsvBuilder.java

```diff
@@ -0,0 +1,167 @@
+package org.traccar.web;
+
+import java.beans.Introspector;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.traccar.helper.Log;
+import org.traccar.model.MiscFormatter;
+
+public class CsvBuilder {
+
+    private static final String LINE_ENDING = "\r\n";
+    private static final String SEPARATOR = ";";
+    private static final DateTimeFormatter DATE_FORMAT = ISODateTimeFormat.dateTime();
+
+    private StringBuilder builder = new StringBuilder();
+
+    private void addLineEnding() {
+        builder.append(LINE_ENDING);
+    }
+    private void addSeparator() {
+        builder.append(SEPARATOR);
+    }
+    public void addLine(String value) {
```

Comment:
It's never supposed to be used outside of the class, so I think we should remove it. There is also not much value even in private methods like this because they are only 2 lines of code each. Can you please remove them.